### ISSUE TITLE
Feature Request: Allow for Deepl-Write

### ISSUE THREAD

**Issue by teertinker** [itagagaki/DeepL-Selected-Text#10]
Hello,
your extension is really useful. Is there a possibility to add an option to switch from deepl translate to deepl write? 

I managed to modify your *background.js* to make it work with deepl write, I simply changed the URL to "https://www.deepl.com/write#" 

Another option would be to create just a new Extension, but i think this would just clutter the system.

Thanks

**Comment by itagagaki** [comment]
Hi,
you want a new context menu and a shortcut key (which one do you like?) to send the selected text to DeepL Write, right?
I'll try it. Maybe I can do it soon.

**Comment by teertinker** [comment]
Hello,
I first thought an option to switch between deepl-write and translate would be sufficient. But are probably use cases, where people want to use both services. Therefore a new context menu sounds reasonable. I never thought about a shortcut key. So no suggestions from this site.

Thanks for the quick response.

**Comment by itagagaki** [comment]
If an extension contains more than one context menu, they are collapsed into a single parent menu. This means there is an extra step to get to the function you want.
So I decided to create an add-on for the DeepL Write separately from this add-on.
Maybe I can publish it in a few days.

**Comment by itagagaki** [comment]
'DeepL-Write for selected text' is now available for download in the ATN.
https://addons.thunderbird.net/addon/deepl-write-for-selected-text/

**Comment by teertinker** [comment]
Thank you very much!

I tested it and it works!

One problem for me was the min. Version in manifest.json
I had to set it to 115 in order to make the extension working on my ubuntu
machine.
(    "strict_min_version": "115.0",)
Since there may be a lot of users with 115ESR still around (Ubuntu does not
provide 128 as deb yet), it could be helpful to lower the min. version in
the source code.

Anyways. Thanks for developing the tool.

Felix
